torchvision/prototype/transforms/_color.py

```diff
@@ -175,33 +152,25 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
             contrast_before=torch.rand(()) < 0.5,
         )
 
-    def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
+    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         if params["brightness"]:
-            input = self._brightness(input)
+            inpt = self._brightness(inpt)
         if params["contrast1"] and params["contrast_before"]:
-            input = self._contrast(input)
+            inpt = self._contrast(inpt)
         if params["saturation"]:
-            input = self._saturation(input)
+            inpt = self._saturation(inpt)
         if params["saturation"]:
-            input = self._saturation(input)
+            inpt = self._saturation(inpt)
         if params["contrast2"] and not params["contrast_before"]:
-            input = self._contrast(input)
+            inpt = self._contrast(inpt)
         if params["channel_shuffle"]:
-            input = self._channel_shuffle(input)
-        return input
+            inpt = self._channel_shuffle(inpt)
+        return inpt
```

Comment:
When possible for future PRs, do such changes on clean up PRs. This will reduce the LOC size for big PRs like this. No need to take action now.